src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RegistrationServiceImplTest {
+    RegistrationService service;
+    User standardUser;
+
+    @BeforeEach
+    void setUp(){
+        service = new RegistrationServiceImpl();
+        standardUser = new User();

```

Comment:
we can initialize service and standardUser in once in @BeforeAll, but leave setter here